config/initializers/sidekiq.rb

```diff
@@ -2,7 +2,7 @@
 # so you don't have to run a separate process.
 # Also benefit from code reloading.
 
-if Rails.env.development? || Rails.env.test?
+if Rails.env.test?
   require "sidekiq/testing"
```

Comment:
Not related to this like. But need to add starting sidekiq process to procfile. Running bin/dev should start the sidekiq and redis process also.

Reply:
> Not related to this like. But need to add starting sidekiq process to procfile. Running bin/dev should start the sidekiq and redis process also.

@danielpaul the linear card didn't specify redis anywhere. sidekiq is already present in the procfile so there was nothing to change.

Reply:
Sorry not Procfile. It's the dev procfile. That's what runs when you do bin/dev. 

https://github.com/danielpaul/RapidRails/blob/main/Procfile.dev

The goal would be that bin/dev running runs everything we need. 

To run the sidekiq process we need redis as well so we'd need to include that also. 

For redis can copy from procfile.dev we have in other projects. 

Add this line before the sidekiq line:


redis: (ps aux | grep 6379 | grep redis | awk '{ print $2 }' | xargs kill -s SIGINT) && redis-server

Reply:
Also the comment line in the start of the file needs to be removed as not relevant since we are doing it in async